.kitchen.docker.yml

```diff
@@ -0,0 +1,28 @@
+#
+# This uses https://rubygems.org/gems/kitchen-docker
```

Comment:
actually you could even use yaml trick to include the default .kitchen.yml file, but that's out of the scope of this PR:)